cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/VisualStudioParser.java

```diff
@@ -122,7 +119,6 @@ private static void collectRangeMeasures(SMInputCursor function, Map<String, Cov
           coverageData.put(sourceId, builder);
         }
 
-        builder.setConditions(startLine - 1, conditions, coveredConditions);
```

Comment:
@Bertk don't understand why this is no more needed?

Reply:
There are 2 issues with this line:
- The calculation ```startLine - 1``` creates measures for wrong code lines
- Microsoft coverage tool only provides block coverage but not branch coverage (_selection statements_). The indicator for **partial covered condition** cannot be set for the relevant code line but only for the code block and the code metric shows more conditions for a source file than conditions exists in the file itself.

A compromise is to remove the calculation and set condition for every block but this is a different way Visual Studio does it and should be avoided.

<img width="1188" alt="motorcontroller-coverage" src="https://user-images.githubusercontent.com/2315215/36639660-ebaf9a1e-1a11-11e8-8010-4d1a9142583a.PNG">

Reply:
The solution with the compromise produces this output:
![partial-covered-block](https://user-images.githubusercontent.com/2315215/36639730-4d16fc7e-1a13-11e8-9b41-3db0bede6e01.PNG)

![motorcontroller-measuredetails](https://user-images.githubusercontent.com/2315215/36639793-4366eab2-1a14-11e8-97ad-72d0aa8264d2.PNG)